docs/hello-neighbour.yaml

```diff
@@ -239,10 +222,12 @@ paths:
           description: "Church question"
           schema:
             $ref: "#/definitions/Question"
+        401:
+          $ref: "#/responses/Unauthenticated"
```

Comment:
We should probably also have 403 here when you try accessing a question which belongs to another church.